### DISCUSSION TITLE
Allow HTML for solutions

### DISCUSSION THREAD

**Discussion by WalterWoshid** [spatie/laravel-ignition#71] [Ideas]
As the title states, would it be possible to allow html for solutions? Or some extension like multiple solutions, or lists or anything like that?

**Comment by freekmurze** [comment]
For security reasons, we're probably not going to allow that.

**Reply by WalterWoshid** [reply]
Hmmmm, understandable, maybe a MultipleSolutions class, which allows to build (complex) combinations to achieve that? I would really love to display a list somehow with multiple solutions
